### DISCUSSION TITLE
copilot not working

### DISCUSSION THREAD

**Discussion by sjava** [community/community#32794] [Copilot Conversations]
I purchased the service of copilot for one year, but I can't use it.
my copilot has disabled.
<img width="762" alt="2022-09-07_11-29" src="https://user-images.githubusercontent.com/26506/189697532-e1410ea0-a50b-45aa-ac5c-d840d71f5cfa.png">

**Comment by thuongtruong1009** [comment]
Hi @sjava 
Currently, Copilot only supports 1 year trial for free accounts. If you register for a drill account but do not use it, they will still be timed as usual. So to use there are 2 methods:
1: You will have to upgrade to a premium account
2: Use edu email for Github account and sign up for Copilot, after 1 year the trial will expire. However, because they are using it on an edu account, so they only notify and extend forever until you no longer use the edu account.
